scst-store/amr/graphql-query.hbs.md

```diff
@@ -3,15 +3,83 @@
 This topic tells you how to connect to the GraphQL playground and how to query each supported data model.
 For information about the AMR data models, see [AMR Data Models](data-model-and-concepts.hbs.md).
 
-## <a id='connecting-to-graphql'></a> Connecting to GraphQL playground
+## <a id='connecting-to-graphql'></a> Connecting to AMR GraphQL
+
+There are two ways you can perform GraphQL queries:
+
+- Using the GraphQL playground
+- Using [cURL](https://curl.se/)
 
 Vmware recommends enabling ingress. The Supply Chain Security
 Tools for Tanzu – Store and Artifact Metadata Repository (AMR) packages share
 the same ingress configuration. Setting `ingress_domain` for the SCST -
-Store applies the same value for AMR. To connect to the AMR GraphQL playground
-when ingress is enabled, go to
-`https://artifact-metadata-repository.<ingress-domain>/play`. You can use this to write and
-execute your own GraphQL queries to fetch data from the AMR.
+Store applies the same value for AMR.
+
+### <a id='amr-graphql-access-token'></a> Retrieving the AMR GraphQL Access Token
+
+Whichever method you choose to access the AMR GraphQL, you first need to retrieve the
+access token.
+
+Use the following command to fetch the token:
+
+```console
+kubectl -n metadata-store get secret amr-graphql-view-token -o json | jq -r ".data.token" | base64 -d
+```
+
+### <a id='connecting-to-graphql-playground'></a> Connecting to AMR GraphQL playground
+
+To connect to the AMR GraphQL playground when ingress is enabled, go to
+`https://artifact-metadata-repository.<ingress-domain>/play`. 
+
+Look for the Headers tab at the bottom of the query window abd add a JSON block containing the auth header 
+in the following format:
+
+```json
+{
+  "Authorization": "Bearer ACCESS-TOKEN"
+}
+```
+
+where:
+
+- `ACCESS-TOKEN` is the AMR GraphQL access token
+
+You can use this to write and execute your own GraphQL queries to fetch data from the AMR.
```

Comment:
❌ 
The token is good only for reading the records and no write. Please change it to

```
You can use this to execute your own GraphQL queries to fetch data from the AMR.

Reply:
Great catch. Thanks!

Reply:
Actually, it does say that. It says you can write and execute your own GraphQL queries, which are used to just fetch data from the AMR.